web/src/main/java/org/wso2/testgrid/web/api/TestPlanService.java

```diff
@@ -174,17 +174,17 @@ public Response getLogContent(@PathParam("id") String id, @QueryParam("truncate"
                                                                 artifactDownloadable.readArtifact(bucketKey);
             return Response.status(Response.Status.OK).entity(truncatedInputStreamData).build();
         } catch (TestGridDAOException e) {
-            String msg = "Error occurred while fetching the TestPlan by id : '" + id + "'";
+            String msg = "Error occurred while fetching the TestPlan by id : '" + id + "' " + e;
```

Comment:
s/e/e.getMessage()/g
(substitue e with e.getMessage!)

Reply:
Fixed in 2c5c954